test/system/005-info.bats

```diff
@@ -182,6 +182,17 @@ host.slirp4netns.executable | $expr_path
     is "$output" ".*graphOptions: {}" "output includes graphOptions: {}"
 }
 
+@test "podman info - additional image stores" {
+    skip_if_remote "--storage-opt flag is not supported for remote"
+    store1=$PODMAN_TMPDIR/store1
+    store2=$PODMAN_TMPDIR/store2
+    mkdir -p $store1 $store2
+    run_podman info --storage-opt=$(podman_storage_driver)'.imagestore='$store1 \
```

Comment:
can you cache `$(podman_storage_driver)` in a var? It seems the function calls podman info each time and info is a rather slow command.